src/styles/index.scss

```diff
@@ -11,4 +11,6 @@
 @import "./components/webchatx.scss";
 
 // pages
-@import "./pages/login.scss";
+
+// components
+@import "./components/webchat";
```

Comment:
plz bring this line to the "components" space  on the top

Reply:
Line removed. Component already present in the Component space